pkg/api/k8s_versions.go

```diff
@@ -39,6 +39,10 @@ const (
 	ciliumCleanStateImageReference                    string = "docker.io/cilium/cilium-init:2018-10-16"
 	ciliumOperatorImageReference                      string = "docker.io/cilium/operator:v1.4"
 	ciliumEtcdOperatorImageReference                  string = "docker.io/cilium/cilium-etcd-operator:v2.0.5"
+	antreaControllerImageReference                    string = "antrea/antrea-ubuntu:latest"
```

Comment:
Antrea 0.2.0 is released today. Could we please update this to antrea/antrea-ubuntu:v0.2.0